### PR TITLE
c#/03.welcome-user Aligning code with Echobot Sample

### PR DESCRIPTION
**Pull request by JasonSowers**

### CODE REVIEW

**Comment by daveta** [comment]
Updates for Program.cs for tracing..

File: samples/csharp_dotnetcore/03.welcome-users/DeploymentScripts/MSbotClone/bot.recipe [hunk 1]
```diff
@@ -4,7 +4,7 @@
     {
       "type": "endpoint",
       "id": "24",
```
Comment (daveta) [inline samples/csharp_dotnetcore/03.welcome-users/DeploymentScripts/MSbotClone/bot.recipe]: Are you generating this?  Kind of interesting your id - I think this is derived from config.bot file..

File: samples/csharp_dotnetcore/03.welcome-users/Startup.cs [hunk 2]
```diff
@@ -110,37 +135,14 @@ public void ConfigureServices(IServiceCollection services)
             });
         }
 
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
```
Comment (daveta) [inline samples/csharp_dotnetcore/03.welcome-users/Startup.cs]: Yeah, this won't show up without changes to Program.cs..

File: samples/csharp_dotnetcore/03.welcome-users/DeploymentScripts/MSbotClone/bot.recipe (hunk 1 above)
Comment (JasonSowers) [reply samples/csharp_dotnetcore/03.welcome-users/DeploymentScripts/MSbotClone/bot.recipe]: I am running the script

**Pull request merged**